apps/pie-microsite/src/assets/styles/trumps/_utilities.scss

```diff
@@ -0,0 +1,7 @@
+@use '../fozzie-config' as f;
+
+.u-hideBelowOrAtWide {
```

Comment:
Would be good to get this into fozzie at some point (we have a lot of these type of [trumps here](https://github.com/justeat/fozzie-components/blob/master/packages/tools/fozzie/src/scss/trumps/_rwd.scss)).

Reply:
yep can do in the near future! Was surprised it wasn't already in there :D